operator/roles/forkliftcontroller/defaults/main.yml

```diff
@@ -101,6 +101,10 @@ api_selfsigned_issuer_name: "{{ api_service_name }}-selfsigned-issuer"
 api_issuer_name: "{{ api_service_name }}-issuer"
 api_certificate_name: "{{ api_service_name }}-selfsigned-ca"
 
+ovirt_populator_image_fqin: "{{ lookup( 'env', 'OVIRT_POPULATOR_IMAGE') or lookup( 'env', 'RELATED_IMAGE_OVIRT_POPULATOR') }}"
```

Comment:
there's a mixture here between the populator and the controller since they use the same image - as we talked offline, I think it wouldn't make sense in our case to have a controller per provider (ovirt, openstack and maybe 'libvirt/kvm' later on), I think we should have one controller, at least until we are able to push that to CDI (to simplify deployments on remote clusters)
but anyway, I think it's better to define two variables here - one for the populator image and one for the controller

Reply:
as we talked offline - having the current state and then split them?